natlas-server/config.py

```diff
@@ -66,7 +66,8 @@ class Config(object):
     # MAIL SETTINGS
     MAIL_SERVER = os.environ.get("MAIL_SERVER", None)
     MAIL_PORT = int(os.environ.get("MAIL_PORT", 587))
-    MAIL_USE_TLS = casted_bool(os.environ.get("MAIL_USE_TLS", False))
+    MAIL_USE_TLS = casted_bool(os.environ.get("MAIL_USE_TLS", True))
+    MAIL_USE_SSl = casted_bool(os.environ.get("MAIL_USE_SSL", False))
```

Comment:
There's a typo in here

Reply:
I don't know what you're talking about https://github.com/natlas/natlas/blob/943b660eafd5198e73c6c6e0f15697ec3a086894/natlas-server/config.py#L70